### PR TITLE
🌱  upgrade to ko v0.10.0

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:
cc: @imjasonh

### CODE REVIEW

**Review by naveensrinivasan — APPROVED** [review]
Thank you!

File: Makefile [hunk 1]
```diff
@@ -191,44 +191,51 @@ ko-build-everything: $(ko-targets)
 
 scorecard-ko:
 	KO_DATA_DATE_EPOCH=$(SOURCE_DATE_EPOCH) KO_DOCKER_REPO=${KO_PREFIX}/scorecard LDFLAGS="$(LDFLAGS)" \
-	ko publish -B \
+	KO_CACHE=/tmp/scorecard ko build -B \
```
Comment (imjasonh) [inline Makefile:194]: I'd probably just have a single shared `KOCACHE`, maybe `/tmp/ko` that just gets shared across all of these.

Comment (developer-guy) [reply Makefile]: I've updated them as /tmp/ko, thank you 🙋🏻‍♂️

**Comment by developer-guy** [comment]
according to @imjasonh feedback to me, I've updated the code.

would you mind reviewing it again please 🙋🏻‍♂️

**Pull request merged**